src/main/java/frc/robot/subsystems/intake/commands/BasicIntakeCommand.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+/**
+ * Default intake command just sucks ball.
+ */
+public class BasicIntakeCommand extends CommandBase {
+    private final Intake intake;
+
+    public BasicIntakeCommand(Intake intake) {
+        this.intake = intake;
+        addRequirements(intake);
+    }
+
+    /**
+     * Set intake motor to an already decided default power.
+     */
+    @Override
+    public void initialize() {
+        intake.setPower(Constants.Intake.POWER);

```

Comment:
Maybe I'm wrong. but I think setting the power of the motor once in `initialize` then not setting it again. won't work (safety feature).

Reply:
I believe it does the motor remembers the last command given, am I missing something?